composer.json

```diff
@@ -16,5 +15,11 @@
   "support": {
     "issues": "https://github.com/Automattic/WPCOM-Legacy-Redirector/issues",
     "source": "https://github.com/Automattic/WPCOM-Legacy-Redirector"
+  },
+  "require-dev": {
+    "automattic/vipwpcs": "^2.2",
+    "dealerdirect/phpcodesniffer-composer-installer": "^0.7.1",
```

Comment:
We can make the constraint here much wider: `^0.4.1 || ^0.5 || ^0.6.2 || ^0.7`.

Reply:
done in fix/travis branch